src/commands/Games/higherLower.ts

```diff
@@ -0,0 +1,267 @@
+import { MessageEmbed } from 'discord.js';
+import { CommandStore, KlasaMessage } from 'klasa';
+import { SkyraCommand } from '../../lib/structures/SkyraCommand';
+import { Events } from '../../lib/types/Enums';
+import { UserSettings } from '../../lib/types/settings/UserSettings';
+import { Time } from '../../lib/util/constants';
+import { LLRCData, LongLivingReactionCollector } from '../../lib/util/LongLivingReactionCollector';
+import { getColor, resolveEmoji } from '../../lib/util/util';
+
+const enum HigherLowerReactions {
+	Higher = 'a:sarrow_up:658450971655012363',
+	Lower = 'a:sarrow_down:658452292558913536',
+	Cancel = ':redCross:637706251257511973',
+	Ok = ':greenTick:637706251253317669',
+	Cashout = '%F0%9F%92%B0' // 💰
```

Comment:
Why not just the emoji as a string?

Reply:
Same reason we use enums for most things, although for here, isnt there a central Emoji  enum instead of having the strings in this file? For example, if we want to use :greenTick: or Cashout in another place, we'll have duplication of the emoji string

Reply:
I meant `Cashout = '💰'`, which would be perfectly valid, save a comment and achieve the exact same.

Reply:
No, it wouldn't.

All emojis are validated using the `escapeURIComponent`:

https://github.com/kyranet/Skyra/blob/1c9accc2ee56e1a8467f5240cf8d82b2fabaf179/src/lib/util/util.ts#L110-L121

So when it gets `'💰'`, it'll be turned into `'%F0%9F%92%B0'`, which would later mismatch.